Input.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Input and Types</title>
+    <link rel="stylesheet" href="input.css" />
+    <link rel="stylesheet" href="component.css">
+  </head>
+  <body>
+    <section class="header"></section>
+    <section>
+      <div class="sidebar">
+        <a href="index.html">Home</a>
+        <a href="Alert.html">Alert</a>
+        <a href="Avatar.html">Avatar</a>
+        <a href="badges.html">Badges</a>
+        <a href="Buttons.html">Buttons</a>
+        <a href="cards.html">Cards</a>
+        <a href="Images.html">Images</a>
+        <a href="Input.html" class="active">Input</a>
+      </div>
+
+      <div class="content">
+        <div class="container">
```

Comment:
I think this container is redundant. The same output can be achieved if this is removed.

Reply:
Yes its updated